src/dfx/Cargo.toml

```diff
@@ -82,6 +82,7 @@ shell-words = "1.1.0"
 slog = { version = "2.5.2", features = ["max_level_trace"] }
 slog-async = "2.4.0"
 slog-term = "2.9.0"
+spinners = "4.1.0"
```

Comment:
Why add a new dependency, rather than using `Environment::new_spinner` ?

Reply:
It's fine to change the spinner package we use, but we should change it, not add another one.

Reply:
I didn't check if we're already using one. I'll drop `spinners` dependency and swap implementation.